### PR TITLE
Fix sqlite plugin windows

### PR DESCRIPTION
**Pull request by SyntevoAlex**
This fixes build of sqlite plugin on Windows.

### CODE REVIEW

File: plugins/NTMakefile [hunk 1]
```diff
@@ -140,7 +146,7 @@ CRAM_FLAGS=/DOBSOLETE_CRAM_ATTR=1
 
 DIGEST_FLAGS=/D "WITH_RC4" /DOBSOLETE_DIGEST_ATTR=1
 
-SCRAM_FLAGS=/DHAVE_SHA512=1
+SCRAM_FLAGS=/DHAVE_SHA256=1
```
Comment (hyc) [inline plugins/NTMakefile:149]: What does this change have to do with SQLite3?

File: plugins/NTMakefile [hunk 2]
```diff
@@ -210,7 +216,7 @@ saslCRAMMD5.dll: $(saslCRAMMD5_objs) saslCRAMMD5.res
 	IF EXIST $@.manifest mt -manifest $@.manifest -outputresource:$@;2
 
 saslDIGESTMD5.dll: $(saslDIGESTMD5_objs) saslDIGESTMD5.res
-	$(LINK32DLL) @<< $(OPENSSL_LIBS) $(LINK32DLL_FLAGS) /out:"saslDIGESTMD5.dll" /implib:"saslDIGESTMD5.lib" $(saslDIGESTMD5_objs) saslDIGESTMD5.res
+	$(LINK32DLL) @<< $(LINK32DLL_FLAGS) /out:"saslDIGESTMD5.dll" /implib:"saslDIGESTMD5.lib" $(saslDIGESTMD5_objs) saslDIGESTMD5.res
```
Comment (hyc) [inline plugins/NTMakefile:219]: What does this change have to do with SQLite3?

**Comment by SyntevoAlex** [comment]
Yes, sorry, these changes slipped in when I was applying patch from other repo which has older SASL.
Normally I double-check diffs, but this time I guess I was too tired or something.
I have now removed all unwanted changes.

**Comment by quanah** [comment]
@hyc looks like this is ready for review now

**Comment by Neustradamus** [comment]
@hyc: Have you seen changes on this PR?

**Comment by hyc** [comment]
Yes. Again, I don't use MSVC, so I can't verify these changes.

**Review by hyc — APPROVED** [review]
I didn't test but the changes look fine.

**Pull request merged**